addon_service/migrations/0001_initial.py

```diff
@@ -239,6 +218,36 @@ class Migration(migrations.Migration):
                 "verbose_name_plural": "External Storage Services",
             },
         ),
+        migrations.CreateModel(
```

Comment:
Will this require a migration rollback / re-migrate when it hits staging?

Reply:
yes, tho staging is on `main` at the moment and there are other breaking migration changes in `develop` already -- i thiiink this PR is the last db-breaking change before it makes sense to freeze the initial migration